### DISCUSSION TITLE
MAC 的電腦遊戲市場

### DISCUSSION THREAD

**Discussion by AngusGuei** [1101-GS3073-C/Python-Community-GS3073-C#423] [General]
應該說大部分人買 MAC 就沒想過要玩遊戲了吧
看 Steam 上兼容的就沒幾個

https://technews.tw/2022/05/18/can-apple-take-the-lead-in-the-ar-gaming-market/

**Comment by lx3211** [comment]
我是本身沒有玩遊戲的習慣所以才買mac的

**Comment by xCk27x** [comment]
最大問題還是廠商同時研發兩種遊戲所需的成本太大
感覺可以直接在Mac裡使用Windows模擬器
之前就是這樣用Mac打英雄聯盟的XD
運作效能也很不賴啦

**Comment by limitx0** [comment]
除了價格以外我就是因為支援的遊戲少才不想買Mac
不過不知道為什麼很多軟體和程式都支援Mac，而且更新速度還很快

**Comment by HsuChen88** [comment]
要用Mac玩遊戲都好麻煩，寧可花點錢去網咖解解饞就好

**Comment by Yilin20021016** [comment]
感覺就是貴在系統跟CPU 但如果慢慢分出來對比其他筆電的話 mca就只有電池跟系統好了

**Comment by Raywei0214** [comment]
我覺得還是看個人需求吧，有些人喜歡有些人不喜歡，有些人就喜歡坐在星巴克之類的開mac裝高尚(沒，又想打遊戲，但前者的的慾望大於後者就買了mac才在哭說好想打遊戲(純屬玩笑

**Comment by awkward-willy** [comment]
或許未來 Cloud Gaming 的技術發展起來， Mac 的使用者也能不受限制的玩到遊戲，當然 Mac 本身就不是遊戲取向的，除此之外還是有其他可取之處。

**Comment by RogelioKG** [comment]
選 Mac 的話通常都是續航取向吧
像是之前極客灣就做過測試：
https://youtu.be/Z0tNtMwYrGA
真的是蠻適合帶出去工作的，
像我的 Windows 筆電，沒插電的話一個小時就掛了。
可以理解為什麼在星巴克看到的都是清一色的 Mac。
